crates/sozo/ops/src/migration/migrate.rs

```diff
@@ -708,6 +708,7 @@ where
     if let Some(world) = &strategy.world {
         ui.print_header("# World");
         ui.print_sub(format!("declare (class hash: {:#x})\n", world.diff.local_class_hash));
+        ui.print_sub(format!("declare (contract address: {:#x})\n", world.contract_address));
```

Comment:
i think we should print `world_address` regardless of the migration strategy since its always present. you take that input from the caller of this function and print it just after line `701`